src/controllers/department.controller.js

```diff
@@ -12,15 +12,16 @@ const { version } = require('chai');
   const addDepartment = async (req ,res ,next) => {
     console.log(req.body);
     try {
-    const validate  = await departmentValidation.validateAsync(req.body);
-    console.log("------------>",validate);
-    const department = await departmentService.addDepartment(validate);
+    const validateDepartment  = await departmentValidation.validateAsync(req.body);
+    console.log("------------>",validateDepartment);
+    const department = await departmentService.addDepartment(validateDepartment);
```

Comment:
you need to pass the actual arguement in sbnake case format. 

suppose you need to pass an object named testObj = {
name: 'atul',
team: backend'
}

than your function call should be like below 
departmentService.addDepartment({name: testObj.name, team: testObj.team})